src/test/java/digital/loom/rhizome/authentication/AuthenticationTest.java

```diff
@@ -70,7 +70,7 @@
             clientId,
             "loom.auth0.com" );
     private static final AuthenticationAPIClient client = auth0.newAuthenticationAPIClient();
```

Comment:
Why do we actually need to make HTTP requests externally to Auth0 inside unit tests?